app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,82 @@
+class TweetController < ApplicationController
+
+  get '/tweets/new' do
+    if !logged_in?
+      redirect "/login"
+    else
+      erb :'tweets/create_tweet'
+    end
+  end
+
+  post '/tweets' do
+    if  logged_in?
+      @tweet = Tweet.new(:content => params["tweet"])
```

Comment:
the best way to handle this would be to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.